sphinx/source/docs/user_guide/interaction/widgets.rst

```diff
@@ -274,10 +320,40 @@ A widget for collecting a line of text from a user:
 .. bokeh-plot:: docs/user_guide/examples/interaction_textinput.py
     :source-position: below
 
-ToggleButton
+More information can be found in the Reference for |TextInput|.
+
+Toggle
 ~~~~~~~~~~~~
```

Comment:
I am surprised this is not an error, Sphinx usually complains when the "underline" length does not exactly match what is right above it. I think we should make it consistent in any case.

Reply:
derp. thanks